nodejs/week3/sqlInjection.txt

```diff
@@ -0,0 +1,8 @@
+With orderByRaw it is possible to execute multiple sql statements. The statement 
```

Comment:
Good job to include a text description in your codebase. As a tip, I would suggest renaming it to README.md, since that is the standard among many repositories.